### ISSUE TITLE
Feature request: UnitPromotions_FakeFeature table 

### ISSUE THREAD

**Issue by Tekamthi** [LoneGazebo/Community-Patch-DLL#10429]
There are two existing tables: UnitPromotions_Terrain and UnitPromotions_Feature

These two tables are more or less identical and include fields like DoubleMove, ExtraMove, attackMod, etc.

Request is to have same but corresponding to the FakeFeatures table (ie rivers).  Presumably some aspect of this is already hard coded in some UAs already

**Comment by n-core** [comment]
This is kinda a weird request because river and lake are not "features", hence Firaxis put it in the `FakeFeatures` table.

**Comment by Tekamthi** [comment]
I understand -- however the FakeFeatures table is very similar to the Features table.

Also, some tables that modify Features recognize entries from both "Features" and "FakeFeatures" -- for example 'Feature_YieldChanges' has entries for both lakes and actual Features table features.

Alternative to having a UnitPromotions_FakeFeatures table, the existing UnitPromotions_Features table could just be made to recognize fake features

being able to use fake features for same kind of unit bonuses/maluses as regular features would allow modder more precision in customizing various implementations

**Comment by azum4roll** [comment]
Anything regarding FakeFeatures (and terrains like TERRAIN_HILL and TERRAIN_MOUNTAIN) are all hardcoded in the DLL. 

The FakeFeatures table is only used in Civilopedia for display.

**Comment by Tekamthi** [comment]
I don't want to add FakeFeatures; specifically I'd like to make movement rules for existing FakeFeatures.  control over CS% in these plots would be nice too, just like the other UnitPromotion_ tables, but immediate plan is to adjust movement

**Comment by azum4roll** [comment]
This falls into the hardcoding in DLL category, regardless of what you do in the database.

**Comment by Tekamthi** [comment]
yes, database adjustments are within my grasp without making request here -- I supposed i don't understand what 'hardcoded' means in this sense.  Its a dll feature that cannot be changed?  If so, no worries, thanks for taking a look.

There is a UA that triggers off rivers for movement (albeit road movement, not double, iirc) -- from this I infer that rivers at least can be identified for the purpose of movement bonus; this one does not appear as database entry either.
